### PR TITLE
Pass the full path to the flake8 config files

### PR DESCRIPTION
**Pull request by gsnedders**

### CODE REVIEW

**Review by jgraham — APPROVED** [review]

File: tools/tox.ini [hunk 1]
```diff
@@ -16,8 +16,8 @@ passenv =
 
 [testenv:py27-flake8]
 deps = -r requirements_flake8.txt
```
Comment (Hexcles) [inline tools/tox.ini]:
```suggestion
deps = -rrequirements_flake8.txt
```

File: tools/tox.ini [hunk 2]
```diff
@@ -16,8 +16,8 @@ passenv =
...
-commands = flake8 --append-config=py27-flake8.ini {posargs}
+commands = flake8 --append-config={toxinidir}/py27-flake8.ini {posargs}
 
 [testenv:py36-flake8]
 deps = -r requirements_flake8.txt
```
Comment (Hexcles) [inline tools/tox.ini]:
```suggestion
deps = -rrequirements_flake8.txt
```

**Review by Hexcles — COMMENTED** [review]
The suggestions I made fixes an unrelated issue.

On my machine with tox 2.5.0, these extra whitespaces cause tox to look for `" requirements_flake8.txt"` which does not exist.

**Review by Hexcles — APPROVED** [review]
LGTM

**Pull request merged**